README.md

```diff
@@ -25,7 +25,26 @@ A transparent, secure proxy for OpenAI's API with token management, rate limitin
 
 ## Quick Start
 
-### Docker (Recommended)
+### Kubernetes with Helm (Recommended for Production)
+
+The Helm chart is published as an OCI artifact to GitHub Container Registry:
+
+```bash
+# Create required secrets
+kubectl create secret generic llm-proxy-secrets \
+  --from-literal=MANAGEMENT_TOKEN="$(openssl rand -base64 32)"
+
+# Install from OCI registry
+helm install llm-proxy oci://ghcr.io/sofatutor/llm-proxy \
+  --version 0.1.0 \
```

Comment:
This hardcodes `--version 0.1.0`. That will go stale immediately and may not exist in GHCR yet. Suggest using `--version <release>` (with an example) or referencing the GHCR package page for available versions.